modules/gapi/src/backends/cpu/gcpuimgproc.cpp

```diff
@@ -180,6 +180,14 @@ GAPI_OCV_KERNEL(GCPUCanny, cv::gapi::imgproc::GCanny)
     }
 };
 
+GAPI_OCV_KERNEL(GCPUGoodFeatures, cv::gapi::imgproc::GGoodFeatures)
+{
+    static void run(const cv::Mat& in, int maxCorners, double quality, double minDist, const cv::Mat& mask, int blockSize, bool useHarris, double k, std::vector<cv::Point2f> &out)
```

Comment:
String too long, split it. Four-five parameters per line for example.

Reply:
Very very minor comment: suggest to preserve variables naming among the G-API operation wrapper, kernel's run method and every other place where you expect to face variables reflecting the same entities (ex: test parameters) to give the user opportunity to perceive the code in different places more easily.

Reply:
splitted; names preserved